cmd/kcn/main.go

```diff
@@ -67,6 +67,8 @@ func init() {
 		// See utils/nodecmd/db_migration.go:
 		nodecmd.MigrationCommand,
 
+		nodecmd.UtilCommand,
```

Comment:
Since other codes have comments, should we add one for this code, too?
```suggestion
		// See utils/nodecmd/util.go:
		nodecmd.UtilCommand,
```

Reply:
Thanks added.